tests/cmds/test_users.py

```diff
@@ -169,6 +226,32 @@ def test_list_users_when_given_excluding_active_and_inactive_uses_active_equals_
     )
 
 
+def test_add_legal_hold_membership_to_user_dataframe_adds_legal_hold_columns_to_dataframe(
```

Comment:
This test is not really needed, `_add_legal_hold_membership_to_user_dataframe` is protected and an implementation detail.

We only really care that when we do `--include-legal-hold-membership` that the CLI outputs that.

Reply:
Removed.